docs/quick-start.md

```diff
@@ -99,7 +99,7 @@ media_folder: "images/uploads" # Media files will be stored in the repo under im
 
 If you're creating a new folder for uploaded media, you'll need to know where your static site generator expects static files. You can refer to the paths outlined above in [App File Structure](#app-file-structure), and put your media folder in the same location where you put the `admin` folder.
 
-Note that the`media_folder` file path is relative to the project root, so the  example above would work for Jekyll, GitBook, or any other generator that stores static files at the project root. It would not, however, work for Hugo, Hexo, Middleman, or others that use a different path. Here's an example that could work for a Hugo site:
+Note that the`media_folder` file path is relative to the project root, so the  example above would work for Jekyll, GitBook or any other generator that stores static files at the project root. It would not, however, work for Hugo, Hexo, Middleman or others that use a different path. Here's an example that could work for a Hugo site:
```

Comment:
I find I default to not using the Oxford comma (too many years writing AP Style for magazine articles), except when the situation is especially confusing without it. I have no issue with using it per se; really a matter of style choice.

Is there a preferred style guide to use, e.g., the Microsoft Manual of Style or Apple Style guide? There are a number of options for technical writing. [ihearttechnicalwriting.com](http://www.ihearttechnicalwriting.com/style-guide-technical-writing/) has a list of several popular ones.